components/apimgt/org.wso2.carbon.apimgt.rest.api.analytics/src/main/java/org/wso2/carbon/apimgt/rest/api/analytics/impl/ApiApiServiceImpl.java

```diff
@@ -0,0 +1,134 @@
+package org.wso2.carbon.apimgt.rest.api.analytics.impl;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wso2.carbon.apimgt.core.api.Analyzer;
+import org.wso2.carbon.apimgt.core.exception.APIManagementException;
+import org.wso2.carbon.apimgt.core.models.analytics.APICount;
+import org.wso2.carbon.apimgt.core.models.analytics.APIInfo;
+import org.wso2.carbon.apimgt.core.models.analytics.APISubscriptionCount;
+import org.wso2.carbon.apimgt.rest.api.analytics.ApiApiService;
+import org.wso2.carbon.apimgt.rest.api.analytics.NotFoundException;
+import org.wso2.carbon.apimgt.rest.api.analytics.dto.APICountListDTO;
+import org.wso2.carbon.apimgt.rest.api.analytics.dto.APIInfoListDTO;
+import org.wso2.carbon.apimgt.rest.api.analytics.dto.APISubscriptionCountListDTO;
+import org.wso2.carbon.apimgt.rest.api.analytics.mappings.AnalyticsMappingUtil;
+import org.wso2.carbon.apimgt.rest.api.common.dto.ErrorDTO;
+import org.wso2.carbon.apimgt.rest.api.common.util.RestApiUtil;
+import org.wso2.msf4j.Request;
+
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+import static org.wso2.carbon.apimgt.rest.api.common.util.RestApiUtil.fromISO8601ToInstant;
+
+public class ApiApiServiceImpl extends ApiApiService {
+
+    private static final Logger log = LoggerFactory.getLogger(ApplicationApiServiceImpl.class);
+
+    /**
+     * Get list of API Info
+     *
+     * @param startTime Filter for start time stamp
+     * @param endTime   Filter for end time stamp
+     * @param createdBy Filter for created user
+     * @param request   MSF4J request
+     * @return API List
+     * @throws NotFoundException When the particular resource does not exist in the system
+     */
+    @Override
+    public Response apiListGet(String startTime, String endTime, String createdBy, Request request) throws
+            NotFoundException {
+        String username = RestApiUtil.getLoggedInUsername(request);
+        startTime = RestApiUtil.fromISO8601ToUTC(startTime);
+        endTime = RestApiUtil.fromISO8601ToUTC(endTime);
+        try {
+            if (log.isDebugEnabled()) {
+                log.debug("Retrieving API information. [From: " + startTime + " to: " + endTime + " created by: " +
+                        "" + createdBy + "]");
+            }
```

Comment:
shall we use one line debug logs using lambdas as in https://garygregory.wordpress.com/2015/09/16/a-gentle-introduction-to-the-log4j-api-and-lambda-basics/